### PR TITLE
Update cgroups information for v2 support

### LINKED ISSUE #16: Document cgroups v2 support
The --apply-cgroups flag can be used to apply cgroups resource and device restrictions on a system using the v2 unified cgroups hierarchy. The resource restrictions must still be specified in the v1 / OCI format, which will be translated into v2 cgroups resource restrictions, and eBPF device restrictions.

### PR DESCRIPTION
**Pull request by dtrudg**
Fixes #16

### CODE REVIEW

File: security.rst [hunk 1]
```diff
@@ -4,28 +4,28 @@
 Security in {Singularity} Containers
 ************************************
 
-Containers are all the rage today for many good reasons. They are light weight, easy to spin-up and require reduced IT management resources as compared to hardware VM environments. More importantly, container technology facilitates advanced research computing by granting the ability to package software in highly portable and reproducible environments encapsulating all dependencies, including the operating system. But there are still some challenges to container security. 
+Containers are all the rage today for many good reasons. They are light weight, easy to spin-up and require reduced IT management resources as compared to hardware VM environments. More importantly, container technology facilitates advanced research computing by granting the ability to package software in highly portable and reproducible environments encapsulating all dependencies, including the operating system. But there are still some challenges to container security.
```
Comment (EmmEff) [inline security.rst:7]:
```suggestion
Containers are all the rage today for many good reasons. They are lightweight, easy to spin-up and require reduced IT management resources as compared to hardware VM environments. More importantly, container technology facilitates advanced research computing by granting the ability to package software in highly portable and reproducible environments encapsulating all dependencies, including the operating system. But there are still some challenges to container security.
```

I hope I'm not nitpicking here.

File: security.rst [hunk 2]
```diff
@@ -4,28 +4,28 @@
...
+to prevent users from escalating privileges once they are inside of a container. The container file system is mounted using the
+``nosuid`` option, and processes are spawned with the ``PR_NO_NEW_PRIVS`` flag. Taken together, this approach provides a secure way
+for users to run containers and greatly simplifies things like reading and writing data to the host system with appropriate
 ownership.
 
-It is also important to note that the philosophy of {Singularity} is *Integration* over *Isolation*. Most container run times strive 
-to isolate your container from the host system and other containers as much as possible. {Singularity}, on the 
-other hand, assumes that the user’s primary goals are portability, reproducibility, and ease of use and that isolation is often a 
-tertiary concern. Therefore, {Singularity} only isolates the mount namespace by default, and will also bind mount several host 
+It is also important to note that the philosophy of {Singularity} is *Integration* over *Isolation*. Most container run times strive
```
Comment (EmmEff) [inline security.rst:25]:
```suggestion
It is also important to note that the philosophy of {Singularity} is *Integration* over *Isolation*. Most container runtimes strive
```

File: security.rst [hunk 3]
```diff
@@ -99,55 +99,57 @@ containers:
...
 cgroups support
 ****************
 
-Starting v3.0, {Singularity} added native support for ``cgroups``, allowing users to limit the resources their containers consume 
-without the help of a separate program like a batch scheduling system. This feature helps in preventing  DoS attacks where one 
-container seizes control of all available system resources in order to stop other containers from operating properly. 
-To utilize this feature, a user first creates a configuration file. An example configuration file is installed by default with 
-{Singularity} to provide a guide. At runtime, the ``--apply-cgroups`` option is used to specify the location of the configuration 
+{Singularity} provides native support for ``cgroups``, allowing users to limit the resources their containers consume
+without the help of a separate program like a batch scheduling system. This feature helps in preventing  DoS attacks where one
```
Comment (EmmEff) [inline security.rst:111]:
```suggestion
without the help of a separate program like a batch scheduling system. This feature helps in preventing DoS attacks where one
```

Extraneous space.

File: security.rst [hunk 4]
```diff
@@ -99,55 +99,57 @@ containers:
...
+{Singularity} supports a number of methods for specifying the security scope and context when running {Singularity} containers.
+Additionally, it supports new flags that can be passed to the action commands; ``shell``, ``exec``, and ``run`` allowing fine
 grained control of security. Details about them are documented `here <https://sylabs.io/guides/\{userversion\}/user-guide/security_options.html>`__.
 
 Security in SCS
 ###############
 
-`Singularity Container Services (SCS) <https://cloud.sylabs.io/home>`_ consist of a Remote Builder, a Container Library, and a 
-Keystore. Taken together, the Singularity Container Services provide an end-to-end solution for packaging and distributing 
+`Singularity Container Services (SCS) <https://cloud.sylabs.io/home>`_ consist of a Remote Builder, a Container Library, and a
```
Comment (EmmEff) [inline security.rst:129]:
```suggestion
`Singularity Container Services (SCS) <https://cloud.sylabs.io>` consist of a Remote Builder, a Container Library, and a
```

**Review by EmmEff — CHANGES_REQUESTED** [review]

File: security.rst (hunk 1 above)
Comment (dtrudg) [reply security.rst:7]: I didn't write this stuff in this PR - it appears to have come in due to the editor removing trailing whitespace. Agree with the changes though.. will add them.

**Review by EmmEff — APPROVED** [review]

**Pull request merged**